hoot-js/src/main/cpp/hoot/js/conflate/ReviewMarkerJs.cpp

```diff
@@ -77,7 +77,7 @@ void ReviewMarkerJs::mark(const FunctionCallbackInfo<Value>& args)
 
   vector<QString> choices = toCpp< vector<QString> >(args[5]);
 
-  ReviewMarker::mark(osmMap, e, note, reviewType, score, choices);
+  ReviewMarker().mark(osmMap, e, note, reviewType, score, choices);
```

Comment:
Wasn't sure about this one...not sure if it ends up getting called in a loop or not so left alone.  If so, maybe should be changed like the others.